src/context/func.rs

```diff
@@ -154,6 +154,16 @@ pub trait FuncCaller: GraphLike + AnalyzerLike<Expr = Expression> + Sized {
                             let cvar = self.add_node(Node::ContextVar(var));
                             ExprRet::Single((ctx, cvar))
                         }
+                        "keccak256" => {
+                            self.parse_ctx_expr(&input_exprs[0], ctx).expect_single();
```

Comment:
I know i dont do this in other parts here, but realistically we shouldn't expect single here - it could be `Fork` variant (but i believe thats the only other variant possible)